integration-test/packages/atoms/for/index.css

```diff
@@ -28,11 +28,11 @@
         border: $(j)px red solid;
       }
 
-      .another-child_$(i)_$(j) {
+      .anotherChild_$(i)_$(j) {
         width: $(j)em;
       }
 
-      &.another-modifier_$(i)_$(j) {
+      &.anotherModifier_$(i)_$(j) {
```

Comment:
How will this effect us?

Reply:
it won't we don't use dashed classes. Ideally we should forbid them breaking the build with an helpful error message